### PR TITLE
Bumping golang version to 1.15 for openshift-controller-manager 4.7, 4.8, and master

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @bparees 
Let's try this again!

File: ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml [hunk 1]
```diff
@@ -3,28 +3,16 @@ base_images:
...
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
 binary_build_commands: make build --warn-undefined-variables
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.15
```
Comment (jupierce) [inline ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml:15]: I recommend you use the same build tag as your builder image -- rhel-8-golang-1.15-openshift-4.7

**Comment by coreydaley** [comment]
/hold
Updating image tag

File: ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml (hunk 1 above)
Comment (coreydaley) [reply ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml]: updated in current push, thanks!

File: ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml [hunk 2]
```diff
@@ -3,28 +3,16 @@ base_images:
...
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-openshift-4.6
 binary_build_commands: make build --warn-undefined-variables
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: rhel-8-golang-1.15-openshift-4.7
```
Comment (bparees) [inline ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml:15]: you need to change the name and namespace fields also.
```
name: builder
namespace: ocp
tag: rhel-8-golang-1.15-openshift-4.7
```

Comment (coreydaley) [reply ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml:15]: updated, thanks

**Comment by bparees** [comment]
looks like the image you're using for build_root now isn't sufficient for running unit tests.

**Comment by coreydaley** [comment]
> looks like the image you're using for build_root now isn't sufficient for running unit tests.

Yeah, ocm pulls in the makefile for build-machinery and apparently it runs that executable for the tests :(

**Comment by coreydaley** [comment]
@bparees ptal

**Pull request merged**